clap-utils/Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2021"
 
 [dependencies]
 chrono = "0.4"
-clap = "2.33.0"
+clap = { version = "3.1.5", features = ["cargo"] }
```

Comment:
Yep here's where we'll need that separate module or crate for clap3, to avoid breaking all downstream users of clap-utils.  

If you do this step first, then it's no longer necessary to migrate all clap-utils users in the monorepo to clap3 at the same time

Reply:
Right, it simplifies life. I've introduced it here -- https://github.com/solana-labs/solana/pull/24096